sdks/python/apache_beam/metrics/metric_test.py

```diff
@@ -172,10 +172,14 @@ def process(self, element):
     # Verify user distribution counter.
     metric_results = res.metrics().query()
     matcher = MetricResultMatcher(
-        namespace='apache_beam.metrics.metric_test.SomeDoFn',
+        step='ApplyPardo',
+        namespace=hc.contains_string('SomeDoFn'),
         name='element_dist',
         committed=DistributionMatcher(
-            sum_value=10, count_value=4, min_value=1, max_value=4))
+            sum_value=hc.greater_than(0),
+            count_value=hc.greater_than(0),
+            min_value=hc.greater_than_or_equal_to(0),
```

Comment:
Why is min different then the others?

I would expect that the same reason this needs to be >= 0 would be the same reason as the others.

Reply:
Fair enough. 

My justification was that as long as there is one single update picked up, then other values should be something non-zero. (e.g. count would be at least one).

Maybe I still underestimated how much uncertainties there is. 

Updated.  Now they are all >=0.